nuxt-frontend/tailwind.config.js

```diff
@@ -36,9 +36,11 @@ module.exports = {
           900: "#3d3d3d",
           87: "#000000DE",
           60: "#00000099",
+          bg: "#121212",
```

Comment:
let's not use it in this way, if it's a shade of the black color then give some meaningful name.